CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+
+
+Create virtual environment and activate it:
+
+```bash
+python3 -m venv .venv
+source ./.venv/bin/activate
+```
+
+Install required libraries
+
+```bash
+pip3 install -r requirements.txt
+pip3 install -r requirements_dev.txt
+```
+
+Run Nifi and Nifi registry on localhost:8080
+```bash
+docker-compose -f resources/docker/latest/docker-compose.yml up
+```
+
+Run the tests
+```bash
```

Comment:
I think you want to run TOX here

Reply:
Also, this project has contributing here : https://github.com/Chaffelson/nipyapi/blob/master/docs/contributing.rst
I'm not sure @Chaffelson would want both.

I also updated contributing in #223

Reply:
ok, I remove my file then